ModelBundle/EventListener/InitialStatusListener.php

```diff
@@ -5,23 +5,36 @@
 use Doctrine\ODM\MongoDB\Event\LifecycleEventArgs;
 use Doctrine\ODM\MongoDB\Event\PostFlushEventArgs;
 use OpenOrchestra\ModelInterface\Model\StatusInterface;
+use Symfony\Component\DependencyInjection\Container;
 
 /**
  * Class InitialStatusListener
  */
 class InitialStatusListener
```

Comment:
you should make this listener extend the class ContainerAware, then the container will be automatically given to the class when it is instanciated